generator/src/test/java/com/scottlogic/deg/generator/generation/RegexStringGeneratorTests.java

```diff
@@ -279,20 +276,22 @@ private void expectOrderedResults(String... expectedValues) {
         StringGenerator generator = constructGenerator(true);
 
         List<String> generatedValues = new ArrayList<>();
-        for (String generatedValue : generator.generateAllValues()) {
-//            System.out.println("generated: " + generatedValue);
-            generatedValues.add(generatedValue);
+        generator.generateAllValues().iterator().forEachRemaining(generatedValues::add);
```

Comment:
Can we use `.collect(Collectors.toList())` might make it a bit easier to read:
```suggestion
        List<String> generatedValues = generator.generateAllValues().stream().collect(Collectors.toList());
```

Reply:
didn't realise it was an Iterator that was returned, thought it was a stream.
No, leave as is, much easier to read than the alternative.